_data/taps/versions/recharge.yml

```diff
@@ -0,0 +1,22 @@
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/integration-versions/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#   [INTEGRATION] VERSIONS   #
+# -------------------------- #
+
+latest-version: "1.0"
```

Comment:
This is my fault for not mentioning it, but going forward, lets use integers (eg: `1` instead of `1.0`) for version numbers. 

This simplifies a lot of Liquid logic that's used throughout the site, and it's easier for users to orient themselves to the right version. I can explain this IRL if you like.